pkg/sql/plan/function/func_builtin.go

```diff
@@ -1123,6 +1123,208 @@ func builtInSerial(parameters []*vector.Vector, result vector.FunctionResultWrap
 	return nil
 }
 
+func BuiltInSerialFull(parameters []*vector.Vector, result vector.FunctionResultWrapper, proc *process.Process, length int) error {
+
```

Comment:
Can we combine serail and serial_full?    Just one function.   The only different is whether return NULL if any arg is null.

The big type switch is tricky, easy to make mistakes.

Reply:
Fixed in this [commit](https://github.com/matrixorigin/matrixone/pull/12093/commits/2d8393489c9ada9f79144c9738e241a1b93ff58e?diff=split&w=0).  @m-schen Kindly review this change.